### ISSUE TITLE
Bash script for automatic deployment

### ISSUE THREAD

**Issue by zanechua** [poowf/invoiceneko#47]

**Comment by zanechua** [comment]
@thelaraman are you a bot? o_o

**Comment by thelaraman** [comment]
Nope, i'm not a bot. See, I can smile ? 😃

**Comment by zanechua** [comment]
@thelaraman doesn't mean you can smile means you're not a bot. :p

**Comment by thelaraman** [comment]
Ok, so now that it is clear I'm not a bot, what do you need in your bash script ?

**Comment by zanechua** [comment]
Basically a deployment script

**Comment by thelaraman** [comment]
Good, so database input, user input ?

**Comment by zanechua** [comment]
Yeah. pretty much. There are some keys that are in there that are unique which requires a user's input. I would get to it but working on getting other features up at the moment.

**Comment by thelaraman** [comment]
Ok, so I will do the 4 points you mentionned, we will see later for user and database inputs.

**Comment by zanechua** [comment]
@thelaraman What's your goal though? To learn?

**Comment by thelaraman** [comment]
Learn ? I'm coding with laravel since 5 years. No man, I'm here to help! Your project seems nice. Relax man, I'm not a bot, I'm not the government. This is github. We are a community. 😉

**Comment by zanechua** [comment]
I'm just wondering. :)

**Comment by thelaraman** [comment]
I have a PR for this
